simple-semigroupoids/simple-semigroupoids.cabal

```diff
@@ -15,12 +15,12 @@ build-type:          Simple
 
 
 library
-  default-language:    Haskell2010
+  default-language:    GHC2021
```

Comment:
is this already supported in 8.10.7?

Reply:
You're right, that's an oversight. I've reverted this change